### PR TITLE
Update bpm-api.md

### PR DESCRIPTION
**Pull request by HappyBonita**
Add limitation on "Get a case variable" to avoid problems

### CODE REVIEW

File: md/bpm-api.md [hunk 1]
```diff
@@ -1850,6 +1850,9 @@ The methods used for this resource are:
       "type":"java.lang.Float"
     }
     ```
+* **Limitations**
+  It is not compatible to render Complexe Data Type variable
```
Comment (rbioteau) [inline md/bpm-api.md]: What is a Complex~~e~~ datatype variable ?

**Comment by HappyBonita** [comment]
or any custom class.

File: md/bpm-api.md (hunk 1 above)
Comment (educhastenier) [reply md/bpm-api.md]: He just meant **complex data type** ~~variable~~

**Comment by rbioteau** [comment]
Data type create this way just generate Java pojo
The variable has not a Complex data type it has a Java data type.
Are you sure that java variable are not supported here ?

File: md/bpm-api.md (hunk 1 above)
Comment (rbioteau) [reply md/bpm-api.md]: You may also use 

```
::: warning _Limitations_  It is not compatible to render Complexe Data Type variable :::
```

To have a pretty warning display in doc.

**Comment by rbioteau** [comment]
I just tried.
It 'works' **but** value contains the the toString value of the java object.
It means that in order to make it work _properly_ you have to implement the toString method of your pojo.
eg: using jackson lib to serialize the object as a JSON string.

IMHO Complex Data type has nothing to do with that and should note be mentionned here.

**Pull request merged**